### ISSUE TITLE
Pagination buttons must have discernible text

### ISSUE THREAD

**Issue by luniki** [opencast/opencast-admin-interface#529]
Currently the arrow buttons (prev, next) don't have any text:

`<button class="button-like-anchor next"></button>`

This violates several WCAG guidelines and has critical impact on blind users.
